docs/README-agent-install.md

```diff
@@ -164,318 +153,303 @@ Virtual service guestshell+ detail
+**Example:** OAC network setup
 
 ~~~
-n7k# virtual-service connect name oac console
+n7k#  virtual-service connect name oac console
 Connecting to virtual-service.  Exit using ^c^c^c
-Trying 127.1.1.5...
-Connected to 127.1.1.5.
-Escape character is '^]'.
-
 
 CentOS release 6.7 (Final)
 Kernel 2.6.99.99 on an x86_64
 
 localhost login: root
-Password: 
+Password: oac
 You are required to change your password immediately (root enforced)
 Changing password for root.
-(current) UNIX password: 
-New password: 
-Retype new password: 
-[root@localhost ~]# 
-~~~
-
-See [References](#references) for more OAC documentation.
+[root@localhost ~]#
 
-#### Set Up OAC Network
+[root@localhost ~]#  chvrf management
 
-The `open agent container` is an independent CentOS container that doesn't inherit settings from NX-OS; thus it requires additional network configuration.  This configuration will be applied inside the `OAC` container.
+[root@localhost ~]#  hostname n7k
 
-Connect to the OAC console, then:
+[root@n7k ~]#  echo 'n7k' > /etc/hostname
 
-~~~bash
-
-# First become root:
-sudo su -
-
-# Enter the management namespace if your device uses the management interface for connectivity.
-chvrf management
-
-# Set up hostname and DNS configuration
-hostname n7k
-
-echo 'n7k' > /etc/hostname
-
-cat >> /etc/resolv.conf << EOF
+[root@n7k ~]#  cat >> /etc/resolv.conf << EOF
 nameserver 10.0.0.202
 domain mycompany.com
-search mycompany.com
 EOF
 ~~~
 
+See [References](#references) for OAC documentation.
+
 ## <a name="agent-config">Puppet Agent Installation, Configuration, and Usage</a>
 
 This section is common to `bash-shell`, `guestshell` and the `open agent container`.
 
-#### Install Puppet Agent
+#### *Step 1. Select and Install the Puppet Agent RPM*
 
-If needed, configure a proxy server to gain network access to `yum.puppetlabs.com`:
+* Optional: Define proxy server variables to allow network access to `yum.puppetlabs.com`:
 
 ~~~bash
 export http_proxy=http://proxy.yourdomain.com:<port>
 export https_proxy=https://proxy.yourdomain.com:<port>
 ~~~
+<br>
 
-Import the Puppet GPG keys.
+* Import the Puppet GPG keys
 
 ~~~
 rpm --import http://yum.puppetlabs.com/RPM-GPG-KEY-puppetlabs
 rpm --import http://yum.puppetlabs.com/RPM-GPG-KEY-reductive
 ~~~
+<br>
+
+* Select the appropriate Puppet RPM for your agent environment
+
+Environment | RPM |
+:--|:--|
+`bash-shell` | <http://yum.puppetlabs.com/puppetlabs-release-pc1-cisco-wrlinux-5.noarch.rpm> |
+`guestshell` | [Puppet defect FACT-1409](https://tickets.puppetlabs.com/browse/FACT-1409): Puppet agent `v1.4.x` is not supported on `guestshell` <p>Use Puppet `v1.3.x` until FACT-1409 is resolved: <http://yum.puppetlabs.com/el/7/PC1/x86_64/puppet-agent-1.3.6-1.el7.x86_64.rpm> <p>Puppet `v1.5.x` will be available at this location after FACT-1409 is resolved: http://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm |
```

Comment:
We should be able to remove this special consideration for the `guestshell` now that puppet-agent-1.5.1 is available for el7 but we should test it first.

http://yum.puppetlabs.com/el/7/PC1/x86_64/